### PR TITLE
glances: update to 3.2.2.

### PR DESCRIPTION
**Pull request by daniel-eys**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/glances/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 pkgname=glances
-version=3.1.7
+version=3.2.2
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-future python3-psutil"
-checkdepends="python3-psutil"
+depends="python3-setuptools python3-future python3-psutil python3-defusedxml"
+checkdepends="python3-psutil python3-defusedxml"
```
Comment (ericonr) [inline srcpkgs/glances/template:8]:
```suggestion
checkdepends="$depends"
```

For simplicity?

Comment (daniel-eys) [reply srcpkgs/glances/template:8]: Sounds good, I'll adjust it this way.

File: srcpkgs/glances/template [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'glances'
 pkgname=glances
-version=3.1.7
+version=3.2.2
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-future python3-psutil"
-checkdepends="python3-psutil"
+depends="python3-setuptools python3-future python3-psutil python3-defusedxml"
```
Comment (paper42) [inline srcpkgs/glances/template:7]: is python3-setuptools really needed? could you check?

Comment (daniel-eys) [reply srcpkgs/glances/template:7]: Indeed, not needed. But `python3-future` pulls it in anyways.

**Pull request merged**